Content.Server/Xenoarchaeology/XenoArtifacts/ArtifactComponent.cs

```diff
@@ -127,22 +127,34 @@ public sealed partial class ArtifactNode : ICloneable
 
     /// <summary>
     /// The trigger for the node
+    /// #IMP: added VVAccess.ReadWrite
     /// </summary>
-    [DataField("trigger", customTypeSerializer: typeof(PrototypeIdSerializer<ArtifactTriggerPrototype>), required: true), ViewVariables]
+    [DataField("trigger", customTypeSerializer: typeof(PrototypeIdSerializer<ArtifactTriggerPrototype>), required: true), ViewVariables(VVAccess.ReadWrite)]
```

Comment:
`VVAccess.ReadWrite` is the default, we can save a tiny bit of space by just deleting `ViewVariables` outright and letting default handle it. same for `Effect` below